tokio/CHANGELOG.md

```diff
@@ -1,3 +1,47 @@
+# 1.6.0 (May 14, 2021)
+
+### Added
+
+- fs: try doing a non-blocking read before punting to the threadpool ([#3518])
+- time: add getter for Interval's period ([#3705])
+- sync: add `mpsc::Sender::{reserve_owned, try_reserve_owned}` ([#3704])
+- sync: add a `MutexGuard::map` method that returns a `MappedMutexGuard` ([#2472])
+- io: add `write_all_buf` to `AsyncWriteExt` ([#3737])
+- net: support non-blocking vectored I/O ([#3761])
+
```

Comment:
```suggestion
- io: implement `AsyncSeek` for `BufReader`, `BufWriter`, and `BufStream` ([#3491])

```

Reply:
Ah, I must not have had that one locally.